src/frontend/src/pages/home/home.tsx

```diff
@@ -1,7 +1,94 @@
 import React from 'react';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import {
+    Link,
+} from "react-router-dom";
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Box from '@material-ui/core/Box';
+
 
 export const Home: React.FC = () => {
     return (
-        <div>Home Page</div>
+        <div>
```

Comment:
```suggestion
        <Box>
```
Perhaps use a Box component instead of the native div element so you can define more stuff using the predefined properties in the API documentation rather than CSS.